docker-gitlabci/build.sh

```diff
@@ -9,7 +9,9 @@ then
 fi
 
 echo "[..] Building Docker image for Gitlab CI..."
-docker build -t domjudge/gitlabci:${VERSION} - <Dockerfile
+cp -r ../docker-contributor/php-config ./
+docker build -t domjudge/gitlabci:${VERSION} . 
```

Comment:
@nickygerritsen for me I cannot build locally without this change, not sure if it has a difference on your machine.

Reply:
Could we use a 
```
COPY ../dokter-contributor/php-config/30-pcov.ini ...
```

Instead? If not this is fine

Reply:
You're more of the Docker expert, but when trying to use an `ADD` the internet indicated that you cannot target things outside of the build context. I forgot to check if `COPY` has the same restrictions.

Reply:
You are right. It is not in the context then

Reply:
Alternative is just having the file 2 times in git, as its small and it will always be ugly, so convenient ugly might be better?

Reply:
I like this better.